lib/util/host-rules.ts

```diff
@@ -174,5 +174,5 @@ export function getAll(): HostRule[] {
 export function clear(): void {
   logger.debug('Clearing hostRules');
   hostRules = [];
-  sanitize.clear();
+  sanitize.clearSanitizedSecretsList();
```

Comment:
Can this be the cause for unsanitized secrets like platform password?

Reply:
Yes, it's part of the problem